src/tools/recipe2plan-cli.ts

```diff
@@ -63,7 +64,9 @@ async function main() {
     Runtime.init('../..');
     fs.mkdirSync(opts.outdir, {recursive: true});
 
-    const plans = await recipe2plan(opts._[0], opts.package);
+    const plans = await Flags.withDefaultReferenceMode(async () => {
+      return await recipe2plan(opts._[0], opts.package);
```

Comment:
optional - I suggest adding an assert to recipe2plan to make sure it is always every called with this flag set to true.

Reply:
That makes sense to me. Changed.